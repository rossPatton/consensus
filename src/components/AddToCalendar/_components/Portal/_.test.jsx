import React from 'react';
import ReactDOM from 'react-dom';
import render from 'react-test-renderer';

import AddToCalendarPortal from '.';

describe('components/AddToCalendar/Portal', () => {
  beforeAll(() => {
    ReactDOM.createPortal = jest.fn((element) => {
      return element;
    });
  });

  afterEach(() => {
    ReactDOM.createPortal.mockClear();
  });

  it('renders without crashing', () => {
    render.create((
      <AddToCalendarPortal
        children={['test']}
        isOpen
        onRequestClose={() => {}}
      />
    ));
  });
});
