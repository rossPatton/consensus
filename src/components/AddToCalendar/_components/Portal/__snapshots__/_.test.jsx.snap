// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/AddToCalendar/Portal renders without crashing 1`] = `
<aside
  className="animated fadeInUp b fixed flex flex-col items-center l p-4 r t-hdr"
>
  <div
    className="relative bg-white border font-semibold p-3 rounded z-10"
  >
    <button
      className="border-0"
      onClick={[Function]}
    />
    <div
      className="font-bold text-3 mb-1"
    >
      Add this meeting to your calendar
    </div>
    <ul>
      <li
        className="mb-1 text-blue-1"
      >
        test
      </li>
    </ul>
  </div>
  <div
    className="absolute b bg-gray-1 l opacity-5 r t"
    onClick={[Function]}
    onKeyPress={[Function]}
    role="button"
    tabIndex={0}
  />
</aside>
`;
